packages/fluentui/docs/src/examples/components/Toolbar/Content/ToolbarExampleMenuRadioGroup.shorthand.tsx

```diff
@@ -1,15 +1,24 @@
-import { Toolbar } from '@fluentui/react-northstar';
+import { Toolbar, ShorthandValue, BoxProps } from '@fluentui/react-northstar';
 import * as React from 'react';
-import { MoreIcon } from '@fluentui/react-icons-northstar';
+import { MoreIcon, UnderlineIcon, FontColorIcon } from '@fluentui/react-icons-northstar';
 
 const ToolbarExampleMenuRadioGroup = () => {
   const [activeIndex, setActiveIndex] = React.useState(0);
   const [menuOpen, setMenuOpen] = React.useState();
 
+  const toolbarItem = (itemName: string, itemIcon: ShorthandValue<BoxProps>) => ({
+    key: itemName,
+    content: itemName,
+    icon: itemIcon,
+    title: itemName,
+  });
+
   return (
     <Toolbar
       aria-label="Toolbar can contain a radio group in a menu"
       items={[
+        toolbarItem('underline', <UnderlineIcon />),
```

Comment:
nit: To be honest, I would just spread objects here. From client's perspective it's much easier to just go trough the items object, instead of seeing what the function that is invoked is doing...

Reply:
Would be much more readable :)

Reply:
changed :)